src/test/java/com/microsoft/azure/vmagent/test/ITAzureVMAgentCleanUpTask.java

```diff
@@ -146,17 +146,5 @@ public void cleanLeakedResourcesRemovesDeployedResources() throws Exception {
                 Assert.assertTrue("Resource shouldn't exist: " + resourceName + " (vmbase: " + depl + " )", resourceName.contains(depl));
             }
         }
-
-        //check the OS disk was removed
```

Comment:
why delete disk related assert?

Reply:
This segments is for unmanaged disk. Since managed disk is default it doesn't work now.